index.js

```diff
@@ -151,10 +151,13 @@ module.exports.processV2 = (output, origin) => {
 };
 
 module.exports.processPlaylist = (output) => {
-  return output.toString().trim().split('\n').map((string) => {
-    const video = JSON.parse(string);
-    return `https://www.youtube.com/watch?v=${video.id}`;
-  });
+  let outputJSON = JSON.parse(output)
+  if (outputJSON["entries"]) {
+    entries = outputJSON["entries"]
+    return entries.map((video) => {
+      return `https://www.youtube.com/watch?v=${video.id}`;
```

Comment:
Can this be made an arrow function expression?

Reply:
Converted to arrow function expression. Kept the brackets around the param to match the pattern for other arrow function expressions in the same file.